core-services/prow/02_config/openshift/coredns-ocp-dnsnameresolver/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  ignore_review_state: true
+  repos:
+  - openshift/coredns-ocp-dnsnameresolver
+  require_self_approval: true
+lgtm:
+- repos:
+  - openshift/coredns-ocp-dnsnameresolver
+plugins:
```

Comment:
Is there a reason we are not including the `restricted_labels`? I don't totally understand this repo, but does our normal NE label process apply to it?

```
label:
  restricted_labels:
    openshift/coredns:
    - allowed_users:
      - candita
      - Miciah
      label: backport-risk-assessed
    - allowed_users:
      - lihongan
      - quarterpin
      - melvinjoseph86
      - ShudiLi
      assign_on:
      - label: backport-risk-assessed
      label: cherry-pick-approved
```

Reply:
I have updated the config for labels as well for the NE process.